public/index.js

```diff
@@ -70,6 +71,7 @@ Vue.createApp({
 				.finally(()=>{
 					this.showInstructions = false;
 					this.isLoading = false;
+					this.timer = setInterval(this.resetState, 180000);
```

Comment:
Looks like `setTimeout` is more appropriate here. The time is 3 minutes. Perhaps a good place to start, but feels a bit long.

What happens if the operator types when the timer fires?

Reply:
Yeah, completely forgot that the timeout function also exists. Swapped it out. Also added a watcher for the input to postpone the timeout if the form is currently submitted.

Reply:
watch is ran whenever the input is focused and a letter is typed?